### PR TITLE
Added pull down to refresh on home tab and animated header

### PR DESCRIPTION
**Pull request by cyburns**

### CODE REVIEW

File: app/(tabs)/_layout.tsx [hunk 1]
```diff
@@ -49,6 +50,7 @@ const Layout = () => {
       <Tabs.Screen
         name="notifications"
         options={{
+          headerShown: false,
```
Comment (avwilliams1995) [inline app/(tabs)/_layout.tsx:53]: good

File: app/(tabs)/index.tsx [hunk 2]
```diff
@@ -1,24 +1,104 @@
...
+    extrapolate: "clamp",
+  });
+
+  scrollY.addListener(({ value }) => {
+    if (value >= 0) {
+      setIsAtTop(false);
+    } else {
+      setIsAtTop(true);
+    }
+  });
```
Comment (avwilliams1995) [inline app/(tabs)/index.tsx:56]: good. Will want to go over to understand a few things on call

File: components/ExploreHeader.tsx [hunk 3]
```diff
@@ -8,45 +14,41 @@ import { FontAwesome } from "@expo/vector-icons";
 
 const ExploreHeader = () => {
   return (
-    <SafeAreaView
-      style={{ flex: 1, backgroundColor: Colors.black, marginBottom: 20 }}
+    <Animated.View
```
Comment (avwilliams1995) [inline components/ExploreHeader.tsx:17]: What;s the difference here with animated view vs SafeArea?

File: components/ProfileHeader.tsx [hunk 4]
```diff
@@ -12,10 +12,11 @@ import { Link } from "expo-router";
 import { Feather } from "@expo/vector-icons";
 import { FontAwesome } from "@expo/vector-icons";
 import { useEffect, useState } from "react";
+import { useSelector } from "react-redux";
+import { RootState } from "@/constants/Types";
 
 const ProfileHeader = () => {
-  const [firstName, setFirstName] = useState("Mark");
-  const [lastName, setLastName] = useState("Zuckerberg");
+  const fullName = useSelector((state: RootState) => state.user.fullName);
```
Comment (avwilliams1995) [inline components/ProfileHeader.tsx:19]: How are we handling name length? Do we have a max input on that? Just curious. Also, how are we handling if a user inputs lets say a link as their name? Need test cases

File: constants/data/fake-listings.json [hunk 5]
```diff
@@ -106,5 +106,94 @@
...
+    "song_title": "Nutshell",
+    "artist": "Alice in Chains",
+    "caption": "If you really want a wild time, pop in the everything playlist on shuffle. you can go from acdc to veggietales to lofi to Kendrick",
+    "city": "Boston",
+    "state": "MA",
+    "review_score": "4.97",
+    "number_of_reviews": "983",
+    "price": "350",
+    "reservation_date": "Sat, 2/24",
+    "table_size": "2-top",
```
Comment (avwilliams1995) [inline constants/data/fake-listings.json:160]: We can ditch some of the old properties

**Review by avwilliams1995 — APPROVED** [review]
Looks good. Will need to go over those edge cases for inputs. At Press, we had bugs early because users would put things like a link or something as their name. Very random just something to tackle earlier than later

File: components/ProfileHeader.tsx (hunk 4 above)
Comment (cyburns) [reply components/ProfileHeader.tsx:19]: check in sign up. Handle this with yup and formik

**Pull request merged**